src/main/java/calculator/InputValue.java

```diff
@@ -0,0 +1,34 @@
+package calculator;
+
+import java.util.List;
+
+public class InputValue {
+    private static final String DEFAULT_DELIMITER = ",:";
+    private final CustomDelimiterExtractor customDelimiterExtractor;
+    private final PositiveIntegerExtractor positiveIntegerExtractor;
+
+    public InputValue(CustomDelimiterExtractor customDelimiterExtractor, PositiveIntegerExtractor positiveIntegerExtractor) {
+        this.customDelimiterExtractor = customDelimiterExtractor;
+        this.positiveIntegerExtractor = positiveIntegerExtractor;
+    }
+
+    public static InputValue of(String value) {
+        String[] strings = value.split("\n");
+        String header = "";
+        String body = strings[0];
+        if (strings.length > 2) {
+            throw new RuntimeException("//;\\n1;2;3 와 같은 형식으로 입력해주세요");
+        }
```

Comment:
strings 의 길이 검증을 header 와 body 객체 생성보다 먼저하면 객체 생성 비용을 아낄 수 있을것 같아요!

```suggestion
        if (strings.length > 2) {
            throw new RuntimeException("//;\\n1;2;3 와 같은 형식으로 입력해주세요");
        }
        String header = "";
        String body = strings[0];
```

Reply:
요 부분은 최종 소스와 중간에 보시는 소스가 달라서 뭔가 좀 커뮤니케이션 미스가 있는 것 같습니다

Reply:
앗 말씀해주셔서 감사합니다!
커밋내역 보고나서 전체 코드로 점검하는 과정에서 확인을 못했네요..